src/main/java/ui/components/NavigableListView.java

```diff
@@ -143,7 +149,7 @@ private void setupKeyEvents() {
                 e.consume();
                 if (selectedIndex.isPresent()) {
                     logger.info("Enter key selection on item " + selectedIndex.get());
-                    onItemSelected.accept(selectedIndex.get());
+                    onItemSelected.accept(selectedIndex.get(), SHOULD_LOAD_BVIEW_ISSUE);
```

Comment:
This line is repeated in a few places. Can abstract into a method with an explanatory name?